common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessage.java

```diff
@@ -20,12 +24,36 @@
  * However remember that one kind of user may be a person with malicious intent so not leak data that should be kept
  * private or implementation details.
```

Comment:
this message can be improved. If I understood it correctly, I'd change it to 
"Note that some users might have malicious intents, therefore keep your data private (or "therefore do not disclose your private data)"

Reply:
yeah it looks more better for me :) thanks!